target/linux/rockchip/rk3399/target.mk

```diff
@@ -0,0 +1,13 @@
+#
+# Copyright (C) 2020 OpenWrt.org
+#
+
+ARCH:=aarch64
+SUBTARGET:=rk3399
+BOARDNAME:=RK3399 boards (64 bit)
+CPU_TYPE:=cortex-a72
```

Comment:
This should most likely be set to ```cortex-a53``` due to the little cluster

Reply:
OK! Done.